schemaregistry/config.go

```diff
@@ -0,0 +1,66 @@
+package schemaregistry
+
+/**
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Config is used to pass multiple configuration options to the Schema Registry client.
+type Config struct {
+	// SchemaRegistryURL determines the URL of Schema Registry
+	SchemaRegistryURL string
```

Comment:
caps for URL does appear to be idiomatic: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
golint complains if you specify `Url`